public/validate-event-form.js

```diff
@@ -0,0 +1,121 @@
+var AddEventForm = document.getElementById('AddEventForm');
+var EditEventForm = document.getElementById('EditEventForm');
+var errorBox = document.getElementById('EventErrorBox');
+
+var englishChars = /[a-zA-Z0-9]/;
+var arabicChars = /[ا-ي0-9]/;
+
+function NotValid(message) {
+  errorBox.innerText = message;
+  window.scrollTo(0, 0);
+}
+
+function Validate(event) {
+  event.preventDefault();
```

Comment:
Okay, one option here would be to only prevent default when the validation fails? Does this make sense? That way you can keep the redirect logic the same on the server.

What do you think?

Reply:
Hmm i would like to to handle server errors on the frontend to avoid redirection, cause also the error messages from the server are currently not really helpful.
i'll just change the error handling